### ISSUE TITLE
A string node that fully ignores richtextformat

### ISSUE THREAD

**Issue by AshtonSparx** [Neos-Metaverse/NeosPublic#2801]
### Is your feature request related to a problem? Please describe.

There's a few times i wanted to make something that gets a slots name, or parse something from an API and it's chunk full of richtext such as changing the size of something, the colour, ect ... and the user didn't end their richtext tag so it messes up what ever is after that text. For example which I find more common in world names

`<color=red><size=2000>WorldName`
then anything after it, is forced to be effected by it.

### Relevant issues

Nothing with "Rich Text"

### Describe the solution you'd like

I would love a logix node that can be fed a string that filters out **any** rich text that is/could be in that string, and it outputs the filtered string.

So `<color=red><size=2000>WorldName`
Just outputs `WorldName`

as an example

### Describe alternatives you've considered

Manually parsing out any/all possible richtext that <i>could</i> be in the string.

### Additional context

![image](https://user-images.githubusercontent.com/53165730/129466254-17ece2d9-5f1f-4bf6-98f4-c3445d9b7bca.png)

Too many times have I seen worlds not do rich text properly and making everything people use look messy. I've even confronted users with the suggestion of end tagging, and have gotten the response "Can't be arsed" -_-

**Comment by Frooxius** [comment]
Are you looking for something to fully strip the rich text? or do you want the output to still have the formatting, just without it affecting what's after it?

There's one thing you can use with the latter, which is the <closeall> (or </closeall>, I haven't checked exactly) tag.

We can add node that strips out any RTF formatting as well though, but then you'll loose all the color and sizes and everything too.

**Comment by AshtonSparx** [comment]
Yeah, I would love a node that strips out any RTF formatting. Losing the colour, size, ect would be the purpose of the node. That would be perfect :D

**Comment by Frooxius** [comment]
Ah yeah we can definitely add that node! We have the functionality internally already, so it's just matter of wrapping it around.

**Comment by AshtonSparx** [comment]
I also didn't know `<closeall>` was a thing! Not sure if having something like that as a node would be worth the while if it's actually/already an RTF
